server/main.go

```diff
@@ -12,6 +12,14 @@ import (
 	"strconv"
 )
 
+<<<<<<< HEAD
```

Comment:
Can you fix the merge conflicts? Just delete the lines you don't want

Reply:
fixed merge conflicts